docs/pages/machine-id/guides/host-certificate.mdx

```diff
@@ -177,7 +177,7 @@ First, create a basic configuration file using the following parameters:
      --join-method=token \
      --certificate-ttl=1h0m0s \
      --ca-pin=(=presets.ca_pin=) \
-     --auth-server=auth.example.com:3025
+     --auth-server=auth.example.com:443
```

Comment:
```suggestion
     --auth-server=proxy.example.com:443
```

Similar to previous comments, let's not confuse readers by giving the impression that auth listens on 443.